detail/pod.vue

```diff
@@ -0,0 +1,137 @@
+<script>
+import { NODE, EVENT } from '@/config/types';
+import createEditView from '@/mixins/create-edit-view';
+import Networking from '@/components/form/Networking';
+import Tab from '@/components/Tabbed/Tab';
+import Container from '@/components/form/Container';
+import Loading from '@/components/Loading';
+import Tabbed from '@/components/Tabbed';
+import NodeScheduling from '@/components/form/NodeScheduling';
+import SortableTable from '@/components/SortableTable';
+import {
+  STATUS, LAST_UPDATED, REASON, MESSAGE, CREATION_DATE
+} from '@/config/table-headers';
+import { mapGetters } from 'vuex';
+
+export default {
+  name: 'PodDetail',
+
+  components: {
+    Networking,
+    Container,
+    NodeScheduling,
+    Tabbed,
+    Tab,
+    Loading,
+    SortableTable
+  },
+
+  mixins: [createEditView],
+
+  async fetch() {
+    const { metadata:{ relationships = [] } } = this.value;
+    const owner = relationships.filter(relationship => relationship.rel === 'owner')[0];
+
+    this.workload = await this.$store.dispatch('cluster/find', { type: owner.fromType, id: owner.fromId });
```

Comment:
resource-instance already has a `getOwners` that does this and handles some problems with this*.  There can be multiple owners, so either handle that or pick one approrpriate workload-typed one from the response.

(*: There are two kinds of 'owner' relationships so just the first one isn't necessarily going to have a 'fromId'.  And without passing `namespace: owner.fromNamespace` you might get the wrong resource I think)